wildfly/server-subsystem/src/main/resources/subsystem-templates/keycloak-infinispan.xml

```diff
@@ -25,7 +25,9 @@
     <supplement name="default">
         <replacement placeholder="CACHE-CONTAINERS">
             <cache-container name="keycloak" jndi-name="infinispan/Keycloak">
-                <local-cache name="realms"/>
+                <local-cache name="realms">
+                    <eviction max-entries="10000" strategy="LRU"/>
```

Comment:
Default value should probably be lower. I just based it on the default value that was hardcoded in InfinispanConnectionProvider.